website/content/en/docs/contribution-guidelines/releasing.md

```diff
@@ -60,38 +67,31 @@ We will use the `v1.3.0` release version in this example.
    by running the
    [deploy-manual](https://github.com/operator-framework/operator-sdk/actions/workflows/deploy-manual.yml)
    GitHub action.  After the image is built, check the security scan
-   results under `Child Manifests`.
-1. A release branch must be created. If you have the proper permissions,
-   you can do this by running the following, assuming the upstream SDK
+   results under `Child Manifests`. Merge the resulting pull requests to update the links.
+1. Make sure that the list of supported OLM versions stated in the [Overview][overview] section of SDK docs is updated. If a new version of OLM needs to be officially supported, follow the steps in [updating OLM bindata](#updating-olm-bindata) section.
+1. Lock down the `master` branch to prevent further commits before the release completes:
+  1. Go to `Settings -> Branches` in the SDK repo.
+  1. Under `Branch protection rules`, click `Edit` on the `master` branch rule.
+  1. In section `Protect matching branches` of the `Rule settings` box, increase the number of required approving reviewers to 6.
```

Comment:
These don't seem to render correctly. At least when I view the file as is they look to be part of the entire list and not indented.
![image](https://user-images.githubusercontent.com/17137/154393520-4f109dcc-6c53-41e1-9c4f-fd1b70a120cd.png)